infra/domain/create_regional_domain_name.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+"""
+This script configures a regional domain name for AWS Apigateway, based on an existing
```

Comment:
The full human-readable name for the service is "API Gateway" and I would prefer it if we used it consistently throughout.

Reply:
fixed.